lib/querystring.js

```diff
@@ -164,20 +174,20 @@ QueryString.escape = function(str) {
   if (lastPos < str.length)
     return out + str.slice(lastPos);
   return out;
-};
+}
 
-var stringifyPrimitive = function(v) {
+function stringifyPrimitive(v) {
   if (typeof v === 'string')
     return v;
   if (typeof v === 'number' && isFinite(v))
     return '' + v;
   if (typeof v === 'boolean')
     return v ? 'true' : 'false';
   return '';
-};
+}
```

Comment:
I dunno, I didn't do anything with this particular function. However, I remember we switched away from that type of usage (no pun intended) because V8 optimizes `typeof xx === yyy` statements. Before we switched to separate `if` statements, [the code was storing the result of `typeof`](https://github.com/nodejs/node/commit/85a92a37ef76059a4733c8e9462ff8da733dfb9e) and that was causing a performance degredation. I suspect `switch(typeof v)` might have the same effect.